components/maker/empty-state/no-apps.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import { Flex, Text, Button } from '@blockstack/ui';
+
+interface NoAppsEmptyStateProps {
+  username: string;
+}
+
+type NoAppsEmptyState = React.FC<NoAppsEmptyStateProps>;
+
+export const NoAppsEmptyState: NoAppsEmptyState = ({ username }) => {
+  const router = useRouter();
+  return (
+    <Flex flexDirection="column" alignItems="center" my={80}>
+      <Text as="h2" mb={6}>
+        You don&apos;t have any apps associated with {username}
+      </Text>
+      <Text mb={6}>
+        Submit your app or sign in with a different Blockstack ID.
+      </Text>
+      <Button onClick={() => router.push('/submit-your-app')}>
```

Comment:
Similar story here with accessibility. It's better to use a `<Link passHref ...>` and `<Button is="a">`. If the new UI library doesn't support this well, we can punt on it for now, but we would need to track that as an issue in `@blockstack/ui`.

Reply:
Button loses styling when using `is="a"`, to investigate